src/sonic_ax_impl/mibs/ietf/rfc3433.py

```diff
@@ -386,15 +388,17 @@ def reinit_data(self):
         if transceiver_dom_encoded:
             self.transceiver_dom = [entry for entry in transceiver_dom_encoded]
 
-        fan_sensor_encoded = Namespace.dbs_keys(self.statedb, mibs.STATE_DB, self.FAN_SENSOR_KEY_PATTERN)
+        # for FAN, PSU and thermal sensors, they are in host namespace DB, to avoid iterating all namespace DBs,
+        # just get data from host namespace DB, which is self.statedb[0].
+        fan_sensor_encoded = self.statedb[0].keys(self.statedb[0].STATE_DB, self.FAN_SENSOR_KEY_PATTERN)
         if fan_sensor_encoded:
             self.fan_sensor = [entry for entry in fan_sensor_encoded]
 
-        psu_sensor_encoded = Namespace.dbs_keys(self.statedb, mibs.STATE_DB, self.PSU_SENSOR_KEY_PATTERN)
+        psu_sensor_encoded = self.statedb[0].keys(self.statedb[0].STATE_DB, self.PSU_SENSOR_KEY_PATTERN)
```

Comment:
Minor suggestion, as state_db[0] is used in multiple places here, will be good to have a constant [ example HOST_NAMESPACE_IDX = 1 ] defined in mibs/__init__.py which could be used in any other implementation as well.